### PR TITLE
Create workflow for CI

### LINKED ISSUE #16: Use GitHub Actions for continuous integration
We've used Travis for other projects, but I want to try GitHub Actions for this repo.

### PR DESCRIPTION
**Pull request by beckyjackson**
This should run any time a push or pull request is made on `master`. It just runs everything in `tests` using python versions 3.6, 3.7, and 3.8.

Resolves #16

### CODE REVIEW

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
The filename `python-package.yml` and the `on` `master` suggests that this was intended for releasing to PyPI, but the content is a CI test. We want both eventually, and as a CI test it works. I would rename this to `python-test.yml`.

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
```
[the remark this answers is not in the document]
Comment (beckyjackson) [reply .github/workflows/python-package.yml:6]: It also runs on pull requests to master, which is why it ran on this branch. Do we want to do more than pull requests to master?

[the remark this answers is not in the document]
Comment (jamesaoverton) [reply .github/workflows/python-package.yml]: I think we want CI to run on every push.

[the remark this answers is not in the document]
Comment (beckyjackson) [reply .github/workflows/python-package.yml]: OK - I've updated this

**Review by jamesaoverton — APPROVED** [review]

**Pull request merged**